### ISSUE TITLE
Dragons Dogma 2 temporary free trial broke prices

### ISSUE THREAD

**Issue by nisehime** [SteamDatabase/SteamDatabase#833]
- _Browser name and version:_ Any
- _Device (mobile or desktop):_ any
- _Page where the issue happens:_ 
https://steamdb.info/app/2054970/
### Describe the bug
Recent free trial for DD2 broke prices for the game on SteamDB main page, now everything is N/A.

**Comment by Nazar1ky** [comment]
Right. SteamDB showing Trial price which N/A. But **you can still access to prices via sub**: https://steamdb.info/sub/734193/

**Comment by xPaw** [comment]
Closing as steamdb basically reflect what steam considers to be the 'price', and that particular game is fine now.
